src/WebSdk/Web/Sdk/Sdk.props

```diff
@@ -23,19 +23,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
   </PropertyGroup>
 
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" Condition="'$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser'" />
+
+  <Import Sdk="Microsoft.NET.Sdk.Razor" Project="Sdk.props" Condition="'$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser'" />
 
-  <Import Sdk="Microsoft.NET.Sdk.Razor" Project="Sdk.props" />
+  <Import Sdk="Microsoft.NET.Sdk.BlazorWebAssembly" Project="Sdk.props" Condition="'$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'browser'" />
 
   <Import Sdk="Microsoft.NET.Sdk.Web.ProjectSystem" Project="Sdk.props" />
 
   <Import Sdk="Microsoft.NET.Sdk.Publish" Project="Sdk.props" />
 
-  <ItemGroup Condition="'$(DisableImplicitFrameworkReferences)' != 'true' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' >= '3.0'">
+  <ItemGroup Condition="'$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser' And '$(DisableImplicitFrameworkReferences)' != 'true' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' >= '3.0'">
```

Comment:
Do you want to store the value of the condition in a property so that the condition check in each of these will look simpler [MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser'?

Reply:
@vijayrkn I was thinking of breaking the contents into two separate files so that way we only check the condition once, and we don't have to litter it over each individual statement. Will that work for you?

Reply:
Ya, that would make it much easier to read/understand what gets imported/conditioned based on the TFM.

Reply:
Ok, I've updated the PR and hopefully will go through the build.